pyrdp/player/ReplayWindow.py

```diff
@@ -27,9 +28,13 @@ def openFile(self, fileName: str, autoplay: bool = False):
         Open a replay file and open a new tab.
         :param fileName: replay path.
         """
-        tab = ReplayTab(fileName)
-        self.mainWindow.addObserver(tab)
+        tab = ReplayTab(fileName, parent=self)
         self.addTab(tab, fileName)
         self.log.debug("Loading replay file %(arg1)s", {"arg1": fileName})
         if autoplay:
             tab.play()
+
+    def resizeEvent(self, event: QResizeEvent):
```

Comment:
I assume this means that the resize event was not being propagated?

Reply:
Well resizing the main window does trigger the resizeEvent of the ReplayWindow because of the layout type used, but it does not trigger the QRemoteDesktop resizeEvent indeed